docs/conf/portland/2021/news/announcing-schedule.rst

```diff
@@ -0,0 +1,61 @@
+
+Start thinking about Writing Day
+--------------------------------
+
+Our pre-conference `Writing Day <https://www.writethedocs.org/conf/portland/{{year}}/writing-day/>`_ will happen on Sunday, and it is much more fun when folks come with their own projects in mind!
+
+If you have a project or are part of a community that you think would benefit from a writing sprint, start brainstorming now about what you might want to focus on.
+If you'd like us to include your project on the Writing Day page, drop us an email at `portland@writethedocs.org <mailto:portland@writethedocs.org>`_.
+Or just show up on Sunday ready to introduce your project!
+
+Shirts on sale
+--------------
+
+The `Write the Docs Portland 2021 Pop-Up Shop <https://teespring.com/wtd-pdx-2021>`_ is still open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want.
+
+The campaign will run in 3-day batches until May 31st. Shipping is available from the USA or Europe. However, due to the global shipping situation, we recommend you order your shirt at least 2-3 weeks in advance if you want to sport your new threads during the conference.
+
+All our events and activities
+-----------------------------
+
+Write the Docs is more than a conference. Every year we make sure to have a variety of activities that our attendees can enjoy, whether you want to learn something, meet new people, or share knowledge. And even though this years conference is held online, we're holding to that plan.
+
+* **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/portland/{{year}}/lightning-talks/>`__ each day after lunch. These are 5 minute presentations and a lot of fun. They can be about anything you want, this is your chance to share something you are passionate about with the audience. Slides are not required.
+* **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/portland/{{year}}/unconference/>`_ all day Monday and on Tuesday afternoon. Want to chat with like-minded folks about a specific thing? This is the place.
+* **Job Fair**. As with the previous few years, on Tuesday morning we'll have a `job fair <https://www.writethedocs.org/conf/portland/{{year}}/job-fair>`_ where you can find your next documentarian friendly job.
+* **Social event**. On Monday night the conference platform will become the social event space. The main stage will be closed, the main focus will be attendee-created sessions. You're welcome to run sessions for board games, trivia quizzes, karaoke, or any other social activity that you want to share with your fellow documentarians.
```

Comment:
```suggestion
* **Social event**. On Monday night we'll be moving to a spatial chat platform for our social event. This will be a more familiar social space where you can move around and join groups of people having conversations. We've used similar platforms at other events, and found that they really help people get to know each other in a more natural way. 
```